src/__tests__/components/Feed.test.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import { Provider } from "react-redux";
+
+import { mount } from "../../enzyme";
```

Comment:
I think that you dont need to import enzyme this way, just use `import { mount } from enzyme";`

Reply:
I remember I did this cause I configure the adapter on this enzyme.js file,  so isnt it really necessary to import from it?

Reply:
These another way to do that by add a setup file for example:

https://github.com/withmoney/withmoney-web/blob/master/tests/test-framework-setup.js#L7

https://github.com/withmoney/withmoney-web/blob/master/tests/jest.config.json#L27

https://github.com/withmoney/withmoney-web/blob/master/package.json#L13

Reply:
finally did it \o/